src/components/EmailSignupForm.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <form
```

Comment:
Uhh not sure why the formatting is showing up like this...it looks alright on my VSCode.